packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/projects/CommitRestControllerIntegrationTests.java

```diff
@@ -76,9 +76,9 @@ public void givenSiriusWebRestAPIWhenWeAskForAllCommitsOfAProjectThenItShouldRet
                 .hasSize(1)
                 .consumeWith(result -> {
                     var restCommits = result.getResponseBody();
-                    assertEquals(TestIdentifiers.UML_SAMPLE_PROJECT, restCommits.get(0).id());
+                    assertEquals(UUID.fromString(TestIdentifiers.UML_SAMPLE_PROJECT), restCommits.get(0).id());
```

Comment:
I think that we shouldn't need to change this file but I need to talk about with @AxelRICHARD first